actors/builtin/miner/miner_state.go

```diff
@@ -841,7 +841,7 @@ func (st *State) AddLockedFunds(store adt.Store, currEpoch abi.ChainEpoch, vesti
 		vestEpoch := quantizeUp(e, spec.Quantization)
 		elapsed := vestEpoch - vestBegin
 
-		targetVest := big.Zero()
+		var targetVest big.Int
```

Comment:
Hmm, I consider this a regression. I recently fixed a NPE caused by this pattern (#382), which is why it looks like this.

What's the linter complaining of? Can we tag this line to survive?

Reply:
The linter complains this is an ineffectual assignment.  We can nolint tag it.